accounts/views.py

```diff
@@ -50,9 +57,19 @@ def form_valid(self, form):
         user = authenticate(email=email, password=password)
         if user.active:
             login(self.request, user)
-        return redirect('accounts/home.html', {'success': True})
+        return redirect('accounts:home')
+
+
+@login_required()
+def edit_user_view(request):
```

Comment:
Spróbuj to zrobić na widoku generycznym. Jeżeli korzystamy z django to wykorzystajmy go na ile się da. :)